apps/src/lab2/views/components/Instructions/instructions.module.scss

```diff
@@ -54,14 +54,24 @@
 
 .bubble {
   position: relative;
-  padding: 12px;
   border-radius: 4px;
   animation: slidein 0.5s;
   background-color: var(--background-neutral-secondary);
   color: var(--text-neutral-primary);
   display: flex;
   flex-direction: column;
-  min-height: 0; 
+  min-height: 0;
+
+  /* If there is scrolling content, it will provide horizontal padding.
+     Doing this allows the scrollbar to use that padding area. */
+  &:has(.scrollingContent) {
+    padding: 12px 0;
+  }
+
+  /* If there is no scrolling content, we will provide horizontal padding. */
+  &:not(:has(.scrollingContent)) {
+    padding: 12px;
+  }
```

Comment:
Do you technically need this rule if we keep the 12px padding in the base `.bubble` class?

Reply:
Oh, I had taken padding out of the base `.bubble` class to avoid having to do an override, figuring we'd always hit one of these two new cases.